sum/sum.py

```diff
@@ -0,0 +1,17 @@
+WORDS = ["a", "neat", "beat", "house", "me", "button", "mat"]
+
+
+def tuple_of_word_lengths():
+    """Return a tuple with the length of the shortest word,
+    the length of the longest word, and the average word length."""
+    word_lengths = ()
+
+    longest_word = max(WORDS, key=lambda x: len(x))
+    shortest_word = min(WORDS, key=lambda x: len(x))
+    average = sum(map(lambda x: len(x), longest_word + shortest_word)) // 2
```

Comment:
I like your eagerness to use composition and `map`! But it make be a bit over-engineered :)

You should be able to just do:

```
average = len(longest_word) + len(shortest_word) // 2
```